stable/owncloud/templates/ingress.yaml

```diff
@@ -1,39 +1,42 @@
 {{- if .Values.ingress.enabled }}
-{{- range .Values.ingress.hosts }}
 apiVersion: extensions/v1beta1
 kind: Ingress
 metadata:
-  name: "{{- printf "%s-%s" .name $.Release.Name | trunc 63 | trimSuffix "-" -}}"
+  name: {{ template "owncloud.fullname" . }}
   labels:
-    app: {{ template "owncloud.fullname" $ }}
-    chart: {{ template "owncloud.chart" $ }}
+    app: {{ template "owncloud.fullname" . }}
+    chart: {{ template "owncloud.chart" . }}
     release: "{{ $.Release.Name }}"
     heritage: "{{ $.Release.Service }}"
   annotations:
+    {{- range .Values.ingress.hosts }}
```

Comment:
this range should be for annotations isn't it?

Reply:
I don't think so, this is only to set that specific annotation in case a host is using TLS, otherwise it could be more annotations outside of the range. The `annotations` field should appear only once.

Reply:
ok! I see it now